js/Genoverse.js

```diff
@@ -318,14 +318,18 @@ const Genoverse = Base.extend({
 
     for (i = 0; i < config.length; i++) {
       track = tracksById[config[i].id];
-      trackFromLibrary = tracksFromLibraryById[config[i].id];
+      trackFromLibrary = tracksFromLibraryById[config[i].defaultId].extend();
```

Comment:
This line fixes filtering of tracks. Problem was that tracks in the library were always mutated by the last item in saved config. That way every filter option were overwritten by last on list. That why it needs new instance when loaded

Reply:
`.extend();`
create new track instance instead manipulating one in library